javascript/javascript2/week2/hyfBay/index.html

```diff
@@ -0,0 +1,80 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
+      </div>
+    </header>
+    <section class="table-view-control">
+      <div class="search">
+        <label>What are you looking for?</label>
+        <input
+          type="search"
+          id="searchName"
```

Comment:
I would suggest using a unique criteria when naming CSS classes and id´s. I have personally always worked with the BEM convention, which splits classes with "-" (search-name). But you can also adopt some other criteria (such as the one used by Airbnb, which is also some kind of industry standard). This could change from company to company, but it is good practice to keep the same standard in a specific project.